besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1816,10 +1816,8 @@ public BesuControllerBuilder setupControllerBuilder() {
     if (DataStorageFormat.BONSAI.equals(getDataStorageConfiguration().getDataStorageFormat())) {
       final DiffBasedSubStorageConfiguration subStorageConfiguration =
           getDataStorageConfiguration().getDiffBasedSubStorageConfiguration();
-      if (subStorageConfiguration.getLimitTrieLogsEnabled()) {
```

Comment:
was this a git merge thing?  odd that we were gating parallel tx on trielog pruning

Reply:
it's a bug I found so I removed this invalid check